backend/pkg/redis/redis.go

```diff
@@ -17,17 +17,28 @@ package redis
 import (
 	"context"
 	"crypto/tls"
+	"errors"
 	"fmt"
 	"net"
 	"net/url"
 	"strconv"
 	"strings"
+	"time"
 
 	"github.com/redis/go-redis/v9"
 )
 
 type Client = redis.Cmdable
 
+const (
+	TemporaryValue           = "TEMP_VALUE"
+	TemporaryValueExpiration = time.Second * 10 // 10s
+)
+
+var (
+	ErrTemporaryCache = errors.New("temporary cache entry")
```

Comment:
I think the naming here is a bit arbitrary.
```suggestion
	ErrCacheInvalid = errors.New("cache invalidated")
```